alembic/env.py

```diff
@@ -5,7 +5,7 @@
 from sqlalchemy import engine_from_config
 from sqlalchemy import pool
 
-from app.db.models import Base
+from app.db.deprecated.models import Base
```

Comment:
@diversemix I'm not sure how this will interact with creating new db models & the multi-schema investigation, so worth looking at carefully! :grin: